EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+shortzzy.*,l.shortzzy.in##.btnlink
```

Comment:
On my end domain is l.top20hub.com

Reply:
What how?

Reply:
![image](https://user-images.githubusercontent.com/8361299/106662999-cabb0800-65ab-11eb-8c62-3888ee4834c6.png)

Reply:
Hmm let me check with other IP.

Reply:
![image](https://user-images.githubusercontent.com/76908410/106681392-58721400-65fb-11eb-95c8-899718a6a4b8.png)
Here!

Reply:
There is anti adblock now in your domian.

Reply:
I don't see anti adblock. Added domain to previous rules to hide ad buttons.

Reply:
@Alex-302 Fake Buttons are back!
![image](https://user-images.githubusercontent.com/76908410/106874924-16d38d00-6711-11eb-8e44-d4f70a7cab11.png)

Reply:
I experienced this before that's why put this rule `##.btnlink`